<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphMenu</class>
 <widget class="QMainWindow" name="GraphMenu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background: #CBD3C8;}

QScrollArea{
	border-radius: 30px;
	background-color: #AEB4A6;
}

QAbstractScrollArea #SWVertices{
	border-radius: 30px;
	background-color: #AEB4A6;
}

QAbstractScrollArea #SWNodes{
	border-radius: 30px;
	background-color: #AEB4A6;
}

QGraphicsView{
	background-color: #CDD1C9;
	border-radius:30px;
	border: 1px solid #000000;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="GVCanva"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QScrollArea" name="SAVertices">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="SWVertices">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>302</width>
             <height>192</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QScrollArea" name="SANodes">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="SWNodes">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>311</width>
             <height>192</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddEllipse"/>
   <addaction name="actionOneDirectionalConection"/>
   <addaction name="actionSelect"/>
   <addaction name="actionSelectVertices"/>
   <addaction name="actionBidirectionalConection"/>
  </widget>
  <action name="actionAddEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ToolBarResources.qrc">
     <normaloff>:/new/Icons/ellipseItemIcon</normaloff>:/new/Icons/ellipseItemIcon</iconset>
   </property>
   <property name="text">
    <string>Add Ellipse</string>
   </property>
  </action>
  <action name="actionOneDirectionalConection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ToolBarResources.qrc">
     <normaloff>:/new/Icons/oneDirectionButtonIcon</normaloff>:/new/Icons/oneDirectionButtonIcon</iconset>
   </property>
   <property name="text">
    <string>One Directional Conection</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ToolBarResources.qrc">
     <normaloff>:/new/Icons/selectionButtonIcon</normaloff>:/new/Icons/selectionButtonIcon</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelectVertices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ToolBarResources.qrc">
     <normaloff>:/new/Icons/selectionOfVerticesIcon</normaloff>:/new/Icons/selectionOfVerticesIcon</iconset>
   </property>
   <property name="text">
    <string>Select Vertices</string>
   </property>
  </action>
  <action name="actionBidirectionalConection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ToolBarResources.qrc">
     <normaloff>:/new/Icons/twoDirectionButton</normaloff>:/new/Icons/twoDirectionButton</iconset>
   </property>
   <property name="text">
    <string>Bidirectional Conection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ToolBarResources.qrc"/>
 </resources>
 <connections/>
</ui>
